arcade/pyproject.toml

```diff
@@ -26,6 +26,7 @@ openai = "^1.36.0" # TODO: relax to an earlier version that still has what we ne
 arcadepy = "~0.2.0"
 pyjwt = "^2.8.0"
 loguru = "^0.7.0"
+tqdm = "^4.67.1"
```

Comment:
This can be relaxed. This is the most recent version. I'm sure previous versions work as well given how simple this addition is.

Reply:
relaxed it